src/js/CliAutoComplete.js

```diff
@@ -193,55 +203,57 @@ CliAutoComplete.builderParseLine = function(line) {
  * Initializes textcomplete with all the autocomplete strategies
  */
 CliAutoComplete._initTextcomplete = function() {
-    var sendOnEnter = false;
-    var self = this;
-    var $textarea = this.$textarea;
-    var cache = self.cache;
+    let sendOnEnter = false;
+    const self = this;
+    const $textarea = this.$textarea;
+    const cache = self.cache;
 
-    var savedMouseoverItemHandler = null;
+    let savedMouseoverItemHandler = null;
 
     // helper functions
-    var highlighter = function(anywhere) {
+    const highlighter = function(anywhere) {
         return function(value, term) {
-            return term ? value.replace(new RegExp((anywhere?'':'^') + '('+term+')', 'gi'), '<b>$1</b>') : value;
+            const anywherePrefix = anywhere ? '': '^';
+            const termValue = value.replace(new RegExp(`${anywherePrefix}(${term})`, 'gi'), '<b>$1</b>');
+            return term ? termValue : value;
         };
     };
-    var highlighterAnywhere = highlighter(true);
-    var highlighterPrefix = highlighter(false);
+    const highlighterAnywhere = highlighter(true);
+    const highlighterPrefix = highlighter(false);
 
-    var searcher = function(term, callback, array, minChars, matchPrefix) {
-        var res = [];
+    const searcher = function(term, callback, array, minChars, matchPrefix) {
+        const res = [];
 
-        if ((minChars !== false && term.length >= minChars) || self.forceOpen || self.isOpen()) {
+        if ((minChars!== false && term.length >= minChars) || self.forceOpen || self.isOpen()) {
```

Comment:
```suggestion
        if ((minChars !== false && term.length >= minChars) || self.forceOpen || self.isOpen()) {
```
Is this intentional?

Reply:
Fixed!